azure-pipelines.yml

```diff
@@ -58,11 +58,11 @@ steps:
     displayName: 'OCaml compiler'
 
   - script: |
-      eval `./opam2_local env`
```

Comment:
Technically, I think you should keep this line. Whenever you use `opam2` you likely want to interact with the custom switch.

Reply:
As long as you aren't using any installed opam products it shouldn't really matter?

Reply:
Correct, but you are going to install `ocamlformat` under whatever switch is active. We should probably also run `eval $(./opam2_local env)` in the rule check script.

Reply:
The rule check script already does this (as it needs it). I have added it back to this switch.